Lib/test/test_ntpath.py

```diff
@@ -905,7 +912,8 @@ def check_error(exc, paths):
         check(['c:/program files/bar', 'C:\\Program Files\\Foo'],
               'c:\\program files')
 
-        check_error(ValueError, ['C:\\Program Files', 'D:\\Program Files'])
+        check_error(ValueError, "Paths don't have the same drive", ['C:\\Program Files', 'D:\\Program Files'])
```

Comment:
Please add also tests for `['C:\\Program Files', 'D:Program Files']` and `['C:Program Files', 'D:\\Program Files']`.

And please break too long lines with `check_error`.